### ISSUE TITLE
Try client out against the iRODs implementation

### ISSUE THREAD

**Issue by ianfore** [ga4gh/fasp-clients#3]
@ianfore's progress on this currently blocked by getting the iRODs implementation running. Have suggestion from Mike to 'docker pull' the images.

**Comment by michael-conway** [comment]
Yes, I -think- this was an error b/c docker-compose build was called, in this case there's no need to build, rather the prebuilt docker images should be all that is required. Pls advise if this isn't the issue. In order to docker-compose build, one would need to first run mvn package on the repo to get the required artifacts.

**Comment by michael-conway** [comment]
...and Andrew had an issue with this so I'm going to take a look, will update in this issue when fixed..
